test/integration/delete_category_test.rb

```diff
@@ -0,0 +1,61 @@
+require "test_helper"
+
+class DeleteCategoryTest < ActionDispatch::IntegrationTest
+
+  setup do
+    admin_user = User.create!(username: "vhodder", email: "vhodder@email.com",
+                              password: "password", admin: true)
+    sign_in_as(admin_user)
+    @category = Category.create!(name: "Sports")
+    @article = Article.create!(title: "Rugby is great", description: "This is a great sport", 
+                                user: admin_user, category_ids: [1])
+  end
+
+  test "should delete category" do
```

Comment:
Ist in diesem Test irgendetwas enthalten, was durch den Test unten nicht abgedeckt ist?
Wir wollen nichts doppelt testen. D.h., dann einfach diesen Test entfernen. :-)